ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -147,6 +154,75 @@ const CromwellButtonRow = (props: {
   );
 };
 
+const RStudioButtonRow = (props: {
```

Comment:
I considered combining this with the Cromwell one, but the number of small differences at the moment would make this pretty messy.

I expect them to converge over time. When I finish the [story to open RStudio](https://precisionmedicineinitiative.atlassian.net/browse/RW-9613) I'll reconsider combining them.

Reply:
makes sense